server/src/main/java/org/elasticsearch/index/mapper/TypeParsers.java

```diff
@@ -37,6 +39,8 @@
 import static org.elasticsearch.common.xcontent.support.XContentMapValues.nodeStringValue;
 
 public class TypeParsers {
+    private static final DeprecationLogger deprecationLogger = new DeprecationLogger(
+        LogManager.getLogger(TypeParsers.class));
```

Comment:
nit: Do we need this newline since it fits under the 140 char limit? I feel like it would be more readable that way

Reply:
I had somehow always thought the limit was 120 :) I'll go ahead and reformat this.